### PR TITLE
Refix for the Docker login with github pkgs.

### PR DESCRIPTION
**Pull request by A-Yamout**
Refix for the Docker login with github pkgs. #151

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,24 +64,19 @@ Try it on [her website!](https://olivia-ai.org)
 ### Installation 
 #### Login to Github 
 
-To get a personal accsess token from Github go to Setings>Developer settings>Personal access tokens
+To get a personal access token from Github go to Setings>Developer settings>Personal access tokens
```
Comment (hugolgst) [inline README.md:67]: I'd put that in a code bracket

File: README.md [hunk 2]
```diff
@@ -64,24 +64,19 @@ Try it on [her website!](https://olivia-ai.org)
...
 
-Click on Genererate new Token and name it you MUST have read and write packages ticked on.
-Then click Generatre new token
+Click on Generate new Token and name it you MUST have read and write packages ticked on.
+Then click Generate new token
 
-Go into home directory 
+Replace TOKEN with the Token that you just made.
```
Comment (hugolgst) [inline README.md:72]: Embrace TOKEN with `

File: README.md [hunk 3]
```diff
@@ -64,24 +64,19 @@ Try it on [her website!](https://olivia-ai.org)
...
-$ cd ~
+$ export PAT=TOKEN
 ```
-Make TOKEN.txt file
-```bash
-$ sudo nano TOKEN.txt
-```
-Paste Token in to file then save
 
 Login to Github (Note: change USERNAME to Gthub username)
```
Comment (hugolgst) [inline README.md:77]: Same here

**Review by hugolgst — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (A-Yamout) [reply README.md:67]: @hugolgst  by code bracket what do you mean?

Comment (hugolgst) [reply README.md:67]:
```
To get a personal access token from Github go to `Setings>Developer settings>Personal access tokens`
```

Comment (A-Yamout) [reply README.md:67]: Ok got it, thank you for the clarification.

File: README.md [hunk 4]
```diff
@@ -64,24 +64,19 @@ Try it on [her website!](https://olivia-ai.org)
 ### Installation 
 #### Login to Github 
 
-To get a personal accsess token from Github go to Setings>Developer settings>Personal access tokens
+`To get a personal access token from Github go to Setings > Developer settings > Personal Access Tokens`
```
Comment (hugolgst) [inline README.md:67]: I'd just put Setings > Developer settings > Personal Access Tokens inside

**Comment by A-Yamout** [comment]
@hugolgst Is that good?

File: README.md (hunk 4 above)
Comment (A-Yamout) [reply README.md:67]: Ok I see what you meant now, sorry for that.

**Comment by A-Yamout** [comment]
@hugolgst Is that good?

**Review by hugolgst — APPROVED** [review]

**Pull request merged**